corehq/apps/zapier/views.py

```diff
@@ -43,14 +40,27 @@ def post(self, request, domain, *args, **kwargs):
             # Return a 409 status code if this criteria isn't met (IE: there is a uniqueness conflict).
             return HttpResponse(status=409)
 
-        ZapierSubscription.objects.create(
-            domain=domain,
-            user_id=str(request.couch_user.get_id),
-            event_name=data['event'],
-            url=data['target_url'],
-            application_id=data['application'],
-            form_xmlns=data['form']
-        )
+        if data['event'] == "new_form":
+            application = Application.get(docid=data['application'])
```

Comment:
nit: Passing the id as a keyword argument isn't strictly necessary, and it would be slightly cleaner to omit it: i.e. `Application.get(data['application'])`